test/core/renderControllerTests.ts

```diff
@@ -0,0 +1,23 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("RenderController", () => {
+  it("Components whose render() is triggered by another Component's render() will be drawn", () => {
+    var link1 = new Plottable.Component();
+    var svg1 = TestMethods.generateSVG();
+    var link2 = new Plottable.Component();
+    var svg2 = TestMethods.generateSVG();
+    link2.renderTo(svg2);
+
+    (<any> link1)._render = () => link2.render();
+    var link2Rendered = false;
```

Comment:
Err the semantics are a bit confusing why is it ordered such that...
```typescript
link2.renderTo(svg2)
```
Doesn't this mean that link2 is rendered, but if so, then why is
```typescript
var link2Rendered = false;
```
directly afterwards?